conf/messages

```diff
@@ -18,7 +18,7 @@ generic.welsh=Cymraeg
 generic.acting_for=You are acting for
 generic.name_not_defined=Name Not Defined
 generic.tax_year_heading=Tax year: April 6 {0} to April 5 {1}
-paye.generic.tax_year_heading=6 April {0} to 5 April {1}
+paye.generic.tax_year_heading=6&nbsp;April&nbsp;{0} to 5&nbsp;April&nbsp;{1}
```

Comment:
Don't have styling in messages file, it's better to replace this in code so you do not have to do this in both english and welsh

Reply:
Agreed 👍 done